agenta-web/src/services/deployment/index.ts

```diff
@@ -0,0 +1,21 @@
+import {Environment} from "@/lib/Types"
+import axios from "@/lib/helpers/axiosConfig"
+import {getAgentaApiUrl} from "@/lib/helpers/utils"
+
+export const fetchEnvironments = async (appId: string): Promise<Environment[]> => {
+    const response = await fetch(`${getAgentaApiUrl()}/api/apps/${appId}/environments/`)
```

Comment:
This is a good time to convert all `fetch` functions with `axios` for consistency.

Reply:
@bekossy are you considering this ?